pyqir-jit/README.md

```diff
@@ -1,22 +1,75 @@
 # pyqir-jit
 
-## Building and Testing
+## Examples
+
+There are JIT examples in the repository:
+- [Bernstein-Vazirani](../examples/jit/bernstein_vazirani.py) ([Bernstein-Vazirani QIR](../examples/jit/bernstein_vazirani.ll))
```

Comment:
both links currently give me a 404. I would also suggest to merge this with the sentence below.

Reply:
There is a mismatch of path resolution between local and on GH. This doc will be moving to a new file to cleanup support for pypi.

Reply:
This will be done in a separate PR.